src/app/progress/PowerCaps.tsx

```diff
@@ -0,0 +1,34 @@
+import { powerLevelByKeyword } from 'app/search/d2-known-values';
+import clsx from 'clsx';
+import { DefaultTFuncReturn, t } from 'i18next';
+import styles from './powercaps.m.scss';
+
+interface PowerCapValue {
+  powercap: keyof typeof powerLevelByKeyword;
+  label: DefaultTFuncReturn | null;
+}
+const caps: PowerCapValue[] = [
+  { powercap: 'powerfloor', label: t('Power Floor') },
+  { powercap: 'softcap', label: t('Soft Cap') },
+  { powercap: 'powerfulcap', label: t('Powerful Cap') },
+  { powercap: 'pinnaclecap', label: t('Pinnacle Cap') },
```

Comment:
You can't use `t` here, because it's going to be evaluated when the module is imported the first time, and it should be reactive to changes in language. Either move this into the component, or use `tl` and translate in the component.

Reply:
Thanks! I've not worked with any i18n stuff before and wasn't sure how it was evaluated. I've moved this into the component now, but didn't see anything change when swapping language. Do I need to add the translations somewhere?

Reply:
I found some other parts of the app that use `t()` and I've now added entries to both `config/i18n.json` and `src/locale/en.json`. Let me know if that looks correct